Source/Editor/Windows/ContentWindow.ContextMenu.cs

```diff
@@ -146,6 +148,148 @@ private void ShowContextMenuForItem(ContentItem item, ref Float2 location, bool
             }
 
             cm.AddSeparator();
+            
+            // Check if is source folder to add new module
+            if (item is ContentFolder sourceFolder && sourceFolder.ParentFolder.Node is ProjectTreeNode node)
```

Comment:
Is this restriction on the ParentFolder being the ProjectTreeNode required?  A lot of the docs, for example, have an editor module created in a sub folder called `Editor` in the `Source` folder.

Reply:
This will only add the context menu option to be added to the source folders right click menu. I figured the best place to create modules is directly under the source folder.